pulp_2to3_migration/app/plugin/rpm/migrator.py

```diff
@@ -121,6 +128,7 @@ def pipeline_stages(self):
             RemoteArtifactSaver(),
             InterrelateContent(),
             RelatePulp2to3Content(),
+            ResolveContentFutures(),
```

Comment:
It can be the very last stage! Great. I hope it's not a typo :)

Reply:
not a typo, just more debugging :dancer: